### PR TITLE
documentation fixes

### PR DESCRIPTION
**Pull request by 7r9s3t5u1v2w4x6y8z0a**
Fixes the spelling mistakes and restructures the documentation

### CODE REVIEW

**Comment by 4yman-el** [comment]
There is already #3. It is merged now.

**Comment by 7r9s3t5u1v2w4x6y8z0a** [comment]
@4yman-el there is a spelling mistake in that and the instructions are incorrect

**Comment by 4yman-el** [comment]
>there is a spelling mistake in that 

We can just fix it.

>the instructions are incorrect

They are inaccurate, not totally wrong.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
-## Linux Toolbox
-A toolbox of utilities for most linux distros
-as fetured on CTT livestream: https://www.youtube.com/watch?v=X1vQ6va5xOc&t=2578s
-## Installation instructions
-1. Clone this repo
-2. Run install.sh
-### MyBash
-
-
+## Chris Titus Tech's Linux Utility
```
Comment (4yman-el) [inline README.md:1]:
```suggestion
## Chris Titus Tech's Linux Utility
A toolbox of utilities for most linux distros as featured on [this CTT livestream](https://www.youtube.com/watch?v=X1vQ6va5xOc&t=2578s).
```

**Review by 4yman-el — COMMENTED** [review]
Simply fix the typo and add the header text (not instructions) to the README

**Comment by 7r9s3t5u1v2w4x6y8z0a** [comment]
@4yman-el merging this PR will fix the issues with the other one, adds a preview image, & adds a one liner to paste into your terminal which is better for clarity etc

Preview of this PR : 
RZ4fsae

**Comment by 4yman-el** [comment]
Okay.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**